docs/INSTALL-COMPLETE-GUIDE.md

```diff
@@ -0,0 +1,145 @@
+
+You'll need a few other things for a one time set up only.
+
+1. Second computer (Linux, Mac or Windows)
+1. USB Mouse and USB Keyboard
+1. Micro SD Card Reader
+1. Screen with HDMI connection
+
+Alternative: If you are able to connect your Raspberry Pi via a wired network interface (LAN), you can set it up via terminal (SSH) only.
+
+## Install Raspberry Pi OS on a Micro SD card
+
+Before you can install the Phoniebox software, you need to prepare your Raspberry Pi and install 
+
+1. Connect your Micro SD card (through a card reader) to your computer
+1. [Download](https://www.raspberrypi.org/software/) the [Raspberry Pi Imager](https://www.raspberrypi.org/blog/raspberry-pi-imager-imaging-utility/) and open it
+1. Select **Raspberry Pi OS** as the operating system (Recommended)
+1. Select your Micro SD card (card will be formatted)
+1. Click `Write`
+1. Wait for the imaging process to be finished (it'll take a few minutes) and eject your SD card
+
+## Initial boot - Set up Wifi connection
+
+1. Connect a USB mouse, a keyboard and a screen through HDMI
+1. Insert the Micro SD card
+1. Start your Raspberry Pi by attaching a power supply
+1. Walk through the "Welcome to Raspberry Pi" wizard (the content of this wizard might have changed after this tutorial was created!)
+    1. Set your locale
+    1. Change your password
+    1. Proceed with with screen settings
+    1. Select your Wifi Network (You are being asked in the Phoniebox install routine whether you want to set up your Wifi again, we will skip this step then)
+    1. Software Update (optional, recommended, takes a while)
+    1. Reboot
+1. Let's enable a few more settings
+    1. Launch `Raspberry Pi Configuration` from the `Preferences` menu
+    1. A window opens with the `System` tab selected
+    1. Select `To CLI` for `Boot` option
+    1. Select `Login as user 'pi'` for `Auto login` option
+    1. Select `Wait for network` for `Network at Boot` option (optional, required for Spotify+ version)
+    1. Navigate to the `Interfaces` tab
+    1. Select `Enabled` next to `SSH`
+    1. Click `OK`
+1. Set up a static IP (optional, recommended)
+    1. Right click on the Wifi sympbol in the upper right corner of your application bar and choose `Wifi & Wired Network Settings`
+    1. Configure `interface` and `wlan0`
+    1. Check `Disabled IPv6` unless you want to provide a static IPv6 address
+    1. Fill out `IPv4` and `Router` (Gateway) options (keep `DNS Servers` and `DNS Search` empty)
+    1. Click `Apply` and `Close`
+1. Optional: If you like, you can **turn off Bluetooth** to reduce energy consumption (unless you want to use any Bluetooth devices with your Phoniebox)
+1. Shutdown your Raspberry Pi (`Application > Logout > Shutdown`)
+
+## Install Phoniebox software
+
+If you want to install the **Spotify+ version**, [read this first](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/develop/docs/SPOTIFY-INTEGRATION.md).
+
+1. While shut down, disconnect mouse, keyboard and HDMI
```

Comment:
There's an issue when using headphone jack and HDMI monitor is connected during installation, this should be addressed here, too.

Reply:
I added this section in the document:

### Using on-board headphone-jack for audio

Note: Installing with an external monitor (HDMI) can create a problem if you use the mini-jack audio out. The problem is that if you plug in a HDMI monitor an additional sound output is added and the index changes. This bothering behavior was introduced, when Raspberry Pi separated headphones jack and HDMI into two different devices in May 2020.

See: [Troubleshooting: headphone audio unavailable after unplugging HDMI](https://github.com/MiczFlor/RPi-Jukebox-RFID/discussions/1300)

If this problem occurs, follow the steps in the next section (configure USB sound card).